test/integration/features/basic.feature

```diff
@@ -0,0 +1,20 @@
+# 
+Feature: Bringup
+  In order to use MiniShift
+  As a user
+  I need to be able to bring up a test environment
+
+  Scenario: Docker environment variables
+    Given a "developer"
+     When executing "minishift start --docker-env=FOO=BAR --docker-env=BAZ=BAT"
+      And should be running
+     Then should have a valid IP address
+     Then executing "minishift ssh echo hello" should contain
+      """
+      hello
+      """
+     Then executing "minishift ssh cat /var/lib/boot2docker/profile" should contain
```

Comment:
> Is this a change we need immediately?

I think so. The plan is to add integration tests now as part of code changes where possible. It makes sense to get the basic use case right. In many case we will have something of the form:

    When executing "minishift <foo>" 
    Then ...
    And ...

And I think in this cases where I execute a minishift command, we will also sometimes not just assert/verify stdout, stderr and exit status. We might have other checks as well, e.g. verifying that config.json has changed.